docs/source/install.ipynb

```diff
@@ -16,7 +16,7 @@
     "pip install evalml\n",
     "```\n",
     "\n",
-    "For older Python versions (3.6.0 in particular), you must install `pip >= 18.0` and `setuptools > 40.0.0` in order to install evalml.\n",
+    "For older Python versions you must install `pip >= 18.0` and `setuptools > 40.0.0` in order to install evalml.\n",
```

Comment:
Do we still need this line? :o Wondering since it seems like from the deletion that we wrote this for 3.6 in particular.